cves/CVE-2011-3192.yml

```diff
@@ -103,10 +107,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
+  answer: |
+    The vulnerability was original identified by Google Security Research, Michal
+    Zalewski in January of 2007. It was eventually exploited in the wild and rose 
+    to fame in 2011 as the Apache Killer.
   date:
```

Comment:
Should be January 2007?

Was this really reported in 2007 and not fixed until 2011? Might be beneficial to add a reference for this.